### PR TITLE
feat(Birthdates): Add plugin for birthday

### PR DESCRIPTION
**Pull request by emilioeduardob**

### CODE REVIEW

File: lita_config.rb [hunk 1]
```diff
@@ -36,6 +36,12 @@
   config.redis[:url] = ENV['BOXEN_REDIS_URL']
   config.handlers.keepalive.url = "http://localhost:#{config.http.port}"
 
+  config.handlers.google_birthdates.calendar_credentials.client_id = ENV.fetch('GOOGLE_CLIENT_ID')
+  config.handlers.google_birthdates.calendar_credentials.client_secret = ENV.fetch('GOOGLE_CLIENT_SECRET')
+  config.handlers.google_birthdates.calendar_credentials.calendar_id = ENV.fetch('GOOGLE_CALENDAR_ID')
+  config.handlers.google_birthdates.calendar_credentials.refresh_token = ENV.fetch('GOOGLE_CALENDAR_REFRESH_TOKEN')
```
Comment (monkeyci) [inline lita_config.rb]: Line is too long. [115/100]

File: lita_config.rb [hunk 2]
```diff
@@ -36,6 +36,12 @@
   config.redis[:url] = ENV['BOXEN_REDIS_URL']
   config.handlers.keepalive.url = "http://localhost:#{config.http.port}"
 
+  config.handlers.google_birthdates.calendar_credentials.client_id = ENV.fetch('GOOGLE_CLIENT_ID')
+  config.handlers.google_birthdates.calendar_credentials.client_secret = ENV.fetch('GOOGLE_CLIENT_SECRET')
+  config.handlers.google_birthdates.calendar_credentials.calendar_id = ENV.fetch('GOOGLE_CALENDAR_ID')
```
Comment (monkeyci) [inline lita_config.rb]: Line is too long. [102/100]

File: lita_config.rb [hunk 3]
```diff
@@ -36,6 +36,12 @@
   config.redis[:url] = ENV['BOXEN_REDIS_URL']
   config.handlers.keepalive.url = "http://localhost:#{config.http.port}"
 
+  config.handlers.google_birthdates.calendar_credentials.client_id = ENV.fetch('GOOGLE_CLIENT_ID')
+  config.handlers.google_birthdates.calendar_credentials.client_secret = ENV.fetch('GOOGLE_CLIENT_SECRET')
```
Comment (monkeyci) [inline lita_config.rb]: Line is too long. [106/100]

**Comment by amosrivera** [comment]
@emilioeduardob line is too long!

**Comment by emilioeduardob** [comment]
🐵 👍 @amosrivera

**Pull request merged**